### PR TITLE
flake/deps: nix 2.19 -> 2.24

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: nixd/lib/Eval/AttrSetProvider.cpp [hunk 1]
```diff
@@ -131,24 +132,25 @@ void fillOptionDescription(nix::EvalState &State, nix::Value &V,
...
       State.forceValue(*It->value, It->pos);
 
       // In nixpkgs some examples are nested in "literalExpression"
       if (nixt::checkField(State, *It->value, "_type", "literalExpression")) {
         R.Example = nixt::getFieldString(State, *It->value, "text");
       } else {
         std::ostringstream OS;
-        It->value->print(State.symbols, OS);
+        // TODO: replace this with something that can traverse the symbol table
+        //It->value->print(State.symbols, OS);
```
Comment (Mic92) [inline nixd/lib/Eval/AttrSetProvider.cpp:153]: @inclyc unsure what to do here. We need symbols to be looked up here in the table. I suppose they should come from `IT` somehow?

**Comment by 573** [comment]
Looks good to me,

**Comment by Mic92** [comment]
> Looks good to me,

Note that I only made it build, haven't tested anything. And also the TODO above is left to be solved.

File: nixd/lib/Eval/AttrSetProvider.cpp (hunk 1 above)
Comment (inclyc) [reply nixd/lib/Eval/AttrSetProvider.cpp]: maybe just:

```C++
It->value->print(State, OS);
```

**Review by inclyc — APPROVED** [review]
Thanks @Mic92! I think this is ready to merge?

**Comment by Mic92** [comment]
> Thanks @Mic92! I think this is ready to merge?

Looks like it?

**Pull request merged**